docs/pages/ecosystem/wallets.mdx

```diff
@@ -1,3 +1,4 @@
 # Wallets that support EIP-5792
 
-* Coinbase Smart Wallet
\ No newline at end of file
+* Coinbase Smart Wallet
+* [Safe{Wallet}](https://github.com/safe-global/safe-wallet-web) – _can connect to dapps via WalletConnect and supports transaction batching natively_
```

Comment:
thanks for adding 🙏looks like you need to escape "{" and "}" here, the build is failing:
`[Safe\{Wallet\}]...`

will merge after updated

Reply:
Thanks!

Reply:
Should be good now 👍